integrations/operator/sidecar/tbot.go

```diff
@@ -240,14 +240,23 @@ func createOrReplaceBot(ctx context.Context, opts Options, authClient auth.Clien
 	if err != nil {
 		return "", trace.Wrap(err)
 	}
+
+	// If the bot already exists, we remove it and its role.
+	// Else, we still check if the role is here. We expect an error (Not Found).
+	// If there's no error, there's a leftover role and we need to remove it to
+	// recover, else the bot creation will fail and the operator be stuck.
+	botRoleName := fmt.Sprintf("bot-%s", opts.Name)
 	if botPresent {
```

Comment:
Can we delete both unconditionally?
And treat `trace.IsNotFound(err)` branch as success

Reply:
Sure, I thought it might be weird in the audit logs to see this, but I guess this is not the nomial case, and users will only hit this for the first operator startup.

Reply:
done